### PR TITLE
Scrum 143

### PR DESCRIPTION
**Pull request by Srasulova**

### CODE REVIEW

File: wombat-admin/src/app/page.tsx [hunk 1]
```diff
@@ -77,6 +80,49 @@ export default function Home() {
...
 
+  // Function to show the status message modal and set a timer to hide it after 5 seconds
+  const showStatusMessage = () => {
+    setStatusMessage(true);
+    setTimeout(() => {
+      setStatusMessage(false);
+    }, 2000);
+  };
+
+  // const updateRequestStatus = async (requestId: string, approvalStatus: 'APPROVED' | 'DENIED') => {
```
Comment (allaboutmike) [inline wombat-admin/src/app/page.tsx:91]: I've mentioned this before, but PLEASE don't commit commented out code. If this was needed before, we would have it in the git history.

File: wombat-admin/src/components/statistics.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function Statistics({ incomingRequests }: StatisticsProps) {
   ).length;
 
   const adminAccessRequiredRequests = incomingRequests.filter(
-    (request) => request.state === "MANUAL_OVERRIDE_REQUESTED"
+    (request) => request.state === "MANUAL_OVERRIDE_ACTIONED"
```
Comment (allaboutmike) [inline wombat-admin/src/components/statistics.tsx:16]: Did we mean to make this change? We want only the requests where they have been actioned?

**Review by allaboutmike — APPROVED** [review]
Overall LGTM. We can delete code, its ok. We have git history if we ever want it back. Also just want to make sure the status change was what we wanted to do.

**Review by NoelCov — APPROVED** [review]
Yup, this looks much better. If you ever need this code back, we can show you how to get it :D

**Pull request merged**